public/static/locales/en/common.json

```diff
@@ -80,9 +81,13 @@
     "timestamp": "{{timestamp}} ago",
     "tools": "Tools",
     "update": "Update",
+    "updateAccountInformation": "Update Account Information",
+    "updateNow": "Update Now",
     "useOldDEForThisFeature": "Please use the <oldDELink>previous version of the Discovery Environment</oldDELink> for access to this feature, until we can add it to this version.",
     "userMenuAriaLabel": "user menu",
     "username": "Username",
+    "userPortalExpiredText": "Your account information is outdated. For continued access to CyVerse services, update your account information now.",
+    "userPortalGracePeriodText": "Update Account Information<br/>We’ve noticed that you haven’t updated your profile in a while.",
```

Comment:
`Update Account Information` needs to be in `<AlertTitle>` tag. So that the heading and message is has different weights.

Reply:
Ah, I didn't notice it was a separate component.  I was looking for a `title` prop in the `Alert` component 👍